docs/src/pages/[platform]/connected-components/storage/storagemanager/examples/StorageManagerEventExample.tsx

```diff
@@ -2,25 +2,60 @@ import * as React from 'react';
 import { StorageManager } from '@aws-amplify/ui-react-storage';
 
 export const StorageManagerEventExample = () => {
-  const [files, setFiles] = React.useState([]);
+  const [files, setFiles] = React.useState({});
 
   return (
     <>
       <StorageManager
         acceptedFileTypes={['image/*']}
-        accessLevel="public"
-        maxFileCount={10}
+        accessLevel="private"
```

Comment:
wouldn't accessLevel private require Authenticator?

Reply:
yes. yes it would.